ratatoskr/app/templates/app/components/schedule_card.html

```diff
@@ -1,8 +1,11 @@
-<div class="card">
+<div class="card card-hover-shadow">
     <div class="card-body">
-        <a href="{% url 'schedule' schedule.id %}">
-            <h5 class="card-title">{{ schedule.owner.socialaccount_set.all.0.extra_data.name }}</h5>
-        </a>    
-        <h6 class="card-subtitle mb-2 text-muted">{{ schedule.name }}</h6>
+        <a href="{% url 'user-schedules' schedule.owner.id %}" class="text-muted text-decoration-none">
+            <div class="d-flex flex-row align-items-center">
+                <img class="me-2 rounded" src="{{ schedule.owner.socialaccount_set.all.0.get_avatar_url }}" alt="{{ schedule.owner.get_full_name }}'s profile picture" style="width: 20px">
+                <h6 class="card-subtitle" style="margin-top: unset !important;">{{ schedule.owner.get_full_name }}</h6>
```

Comment:
Don't really see a reason to use an inline style here, could have just added the `mt-0` class here. If you need a refresher on bootstrap spacing modifiers you can look here: https://getbootstrap.com/docs/5.0/utilities/spacing/#notation. Also, when looking at boostrap documentation, remember to check if it has 5.0 in the URL and not an older version.

Reply:
Unless the default margin-top was something you wanted, then you can just ignore this